internal/application/application.go

```diff
@@ -184,29 +136,8 @@ func WorkingDirectory(app *redskyappsv1alpha1.Application) string {
 	return ""
 }
 
-// LoadResources loads all of the resources for an application, using the supplied file system
-// to load file based resources (if necessary).
-func LoadResources(app *redskyappsv1alpha1.Application, _ filesys.FileSystem) (resmap.ResMap, error) {
-	kf := scan.NewKonjureFilter(WorkingDirectory(app), nil)
-	kf.KeepStatus = false
-
-	var buf bytes.Buffer
-	err := kio.Pipeline{
-		Inputs:  []kio.Reader{app.Resources},
-		Filters: []kio.Filter{kf},
-		Outputs: []kio.Writer{&kio.ByteWriter{Writer: &buf}},
-	}.Execute()
-	if err != nil {
-		return nil, err
-	}
-
-	dp := provider.NewDefaultDepProvider()
-	f := resmap.NewFactory(dp.GetResourceFactory(), dp.GetConflictDetectorFactory())
-	return f.NewResMapFromBytes(buf.Bytes())
-}
-
 func cleanName(n string) string {
-	return strings.Map(func(r rune) rune {
+	n = strings.Map(func(r rune) rune {
```

Comment:
not part of this PR, but while looking through this func, is `.` really valid? I think I would have expected this to fail for dns reasons.

Reply:
True story.